src/layout.css

```diff
@@ -12,17 +12,17 @@
  */
 
 /**
- * Grids require at minimum two elements: A parent with the class `grid` and a child with the class `col`. By default, each `col` in a grid will occupy equal space. The `grid` class includes `*-mm`, `*-ml`, and `*-mxl` variations to target screen sizes.
+ * Grids require at minimum two elements: A parent with the class `grid` and a child with the class `col`. To define the width of `col` in a grid, add a width class, `col--auto`, or both. The `grid` class includes `*-mm`, `*-ml`, and `*-mxl` variations to target screen sizes.
  *
- * When using these classes, do not apply left or right margin or padding directly to the \`grid\` or \`col\` elements or you will face unexpected side-effects.
+ * When using these classes, do not apply left or right margin or padding directly to the `grid` or `col` elements or you will face unexpected side-effects.
```

Comment:
Is this statement still true? The %-based margin examples in `scripts/build-layout-scales.js` use a `col` and margin class. Should we update the margin class example so that we are not applying the margin on the `col`?

Reply:
It's not accurate, never was. Looking at: https://github.com/mapbox/assembly/blob/publisher-staging/scripts/build-layout-scales.js#L79-L103

The advice should be:
- User should not apply margin to `grid`
- User should not apply padding to `col`